CHANGELOG.md

```diff
@@ -5,6 +5,10 @@
 * [ENHANCEMENT]
 * [BUGFIX]
 
+## 0.7.1 / 2023-06-12
```

Comment:
Ah, one comment I just noticed.
The date of the changelog entry is wrong, should be `2023-07-12`.

And thanks for releasing a fix so quickly!